src/playwright/pages/event.page.ts

```diff
@@ -0,0 +1,213 @@
+import type { Locator, Page } from '@playwright/test';
+
+import { Language } from '../../types';
+import { LANGUAGES } from '../constants';
+import { expect } from '../testWithFixtures';
+import { BasePage } from './base.page';
+import {
+  MOCK_ENROL_OCCURRENCE_MUTATION_RESPONSE,
+  MOCK_EVENT_QUERY_RESPONSE,
+  MOCK_SCHOOLS_AND_KINDERGARTENS_LIST_QUERY_RESPONSE,
+  MOCK_STUDY_LEVELS_QUERY_RESPONSE,
+} from '../mocks';
+import { TRANS, TranslationsOf } from '../translations';
+import { GetByRoleOptions } from '../types';
+
+type EnrolmentFormButton = TranslationsOf<'studyLevel' | 'submitEnquiry'>;
+type EnrolmentFormTextBox = TranslationsOf<
+  | 'additionalInfo'
+  | 'emailAddress'
+  | 'group'
+  | 'name'
+  | 'phoneNumber'
+  | 'studyGroupUnit'
+>;
+type EnrolmentFormSpinButton = TranslationsOf<'adults' | 'children'>;
+type EnrolmentFormCheckBox = TranslationsOf<'acceptDataSharing'>;
+
+type OccurrencesSectionButton = TranslationsOf<
+  'showOccurrenceDetails' | 'reservationEnquiry'
+>;
+
+type StudyLevelOption = TranslationsOf<'preschoolStudyLevel'>;
+type StudyGroupUnitOption = TranslationsOf<'daycareKamppiStudyGroupUnit'>;
```

Comment:
If the file becomes too huge and hard to read, I think we could use /pages/events subfolder and have the content of this file splitted into multiple files under that subfolder. So far, I think it's okay to have all this here in a single file. Most likely this file is "ready" after this PR and won't change much in future anymore.

Reply:
Moved translations to page object model files in 3dabd38fc29cdb53ea26a96822fc0dc58fa3091e